src/GovUk.Education.ExploreEducationStatistics.Content.Model/KeyStatistic.cs

```diff
@@ -1,9 +1,54 @@
-namespace GovUk.Education.ExploreEducationStatistics.Content.Model
+#nullable enable
+using System;
+using AutoMapper;
+using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using Newtonsoft.Json;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Model;
+
+public abstract class KeyStatistic : ICreatedUpdatedTimestamps<DateTime, DateTime?>
 {
-    public class KeyStatistic
+    public Guid Id { get; set; }
+
+    public Guid ReleaseId { get; set; }
+
+    [JsonIgnore, IgnoreMap] public Release Release { get; set; } = null!;
```

Comment:
Do we need JsonIgnore here?  Are we intentionally transforming this into JSON at some point?

Reply:
I don't think so - I think this is a hangover from before I created `KeyStatisticViewModel`.

I've removed it.